iib/web/models.py

```diff
@@ -241,6 +241,15 @@ def get_or_create(cls, name):
         return operator
 
 
+class RequestBuildTag(db.Model):
```

Comment:
@midnightercz  I believe we talked about creating an association table here. `RequestBuildTag` should only be associating the two tables. Like `RequestArchitecture` on line 158. 

Another table called the `BuildTag` should house the `id` and `name` like the `Architecture` table. Could you please change it?

Reply:
I get your point, but unlike architecture which can be chosen only from limited set of choices, build tag can be any string and most probably even will be unique for every request. Does it make sense then to create enumeration table and association table?

Reply:
It would be nice to add a `__repr__` function too.